backend/pkg/repository/groupMemberRepository.go

```diff
@@ -41,7 +41,6 @@ func (r *GroupMemberRepository) RemoveMemberFromGroup(groupId, userId int) error
 
 // CreateGroupInvitation creates a new invitation in the database.
 // It returns an error if any.
-// TODO: implement notifications for group invitations
 func (r *InvitationRepository) CreateGroupInvitation(invitation model.GroupInvitation) error {
 	query := `INSERT INTO group_invitations (group_id, user_id, status) VALUES (?, ?, ?)`
 	_, err := r.db.Exec(query, invitation.GroupId, invitation.JoinUserId, &invitation.InviteUserId, "pending")
```

Comment:
this will throw error because query has 3 values but r.db.Exec gives 4 parameters. 
query := `INSERT INTO group_invitations (group_id, join_user_id, invite_user_id, status) VALUES (?, ?, ?, ?)`